Sources/Datadog/OpenTracing/OTSpan.swift

```diff
@@ -74,3 +74,56 @@ public extension OTSpan {
         self.finish(at: Date())
     }
 }
+
+/// Error conveniences
+public extension OTSpan {
+    /// Set or replace the error for the given span.
+    /// This is a convenience to set the proper tags with error details. Consider the `setError(message:stacktrace:file:line:)` variant for a better control over the error details.
+    ///
+    /// - parameter error: An object conforming to the `Error` protocol.
+    /// - parameter file: A string identifying the file where the `Error` was caught. The default is `#fileID` which means `ModuleName/Filename.extension`, consider an helpful yet concise identifier when overriding the default.
+    /// - parameter line: The line number in the file where the `Error` was caught.
+    func setError(
+        _ error: Error,
+        file: StaticString = #fileID,
```

Comment:
Nice, I didn't know this new `#fileID`. Regarding the [Expressions doc](https://docs.swift.org/swift-book/ReferenceManual/Expressions.html) I think we may want to apply it wider in our codebase. WDYT about creating separate JIRA to incorporate this change in other places in the future?

Reply:
I think that it's a great idea to do a follow-up pass on this. I'll create a JIRA for that!

Reply:
This is tracked by RUMM-1035.